widgetssdk/src/test/java/com/glia/widgets/chat/controller/ChatControllerTest.kt

```diff
@@ -64,6 +67,8 @@ import org.mockito.kotlin.verify
 import org.mockito.kotlin.whenever
 import org.robolectric.RobolectricTestRunner
 
+private const val s = "CompanyName"
```

Comment:
Although it's a unit test, I would suggest using a meaningful name instead of "s".

Reply:
I have no idea what it is :)